resources/guest_configs/patches/0002-ci-adjust-6.1-guest-kernel-configs-for-our-CI.patch

```diff
@@ -0,0 +1,76 @@
+From 911b916fdc53acfc7229302b029fd615f1e670cf Mon Sep 17 00:00:00 2001
```

Comment:
[commenting here since GH does not let me comment on commit title/body]

typo in commit title: updsate